docs/library/jesd204/axi_jesd204_tx/index.rst

```diff
@@ -0,0 +1,1007 @@
+
+   * - Name
+     - Description
+   * - :git-hdl:`axi_jesd204_tx.v 
+       <library/jesd204/axi_jesd204_tx/axi_jesd204_tx.v>`
+     - Verilog source for the peripheral.
+   * - :git-hdl:`axi_jesd204_tx_ip.tcl
+       <library/jesd204/axi_jesd204_tx/axi_jesd204_tx_ip.tcl>`
+     - TCL script to generate the Vivado IP-integrator project for the
+       peripheral.
+
+Block Diagram
+-------------
+
+.. image:: axi_jesd204_tx_204c.svg
+   :scale: 100%
+   :align: center
+
+Synthesis Configuration Parameters
+----------------------------------
+
+.. role:: raw-html(raw)
+   :format: html
+
+.. hdl-parameters::
+   :path: library/jesd204/axi_jesd204_tx
+
+   * - ID
+     - Instance identification number.
+   * - NUM_LANES
+     - Maximum number of lanes supported by the peripheral.
+   * - NUM_LINKS
+     - Maximum number of links supported by the peripheral.
+   * - LINK_MODE
+     - |  Decoder selection of the link layer.
+       |  1 - 8B/10B mode;
+       |  2 - 64B/66B mode.
+   * - DATA_PATH_WIDTH
+     - Data path width in bytes. Set it 4 in case of 8B/10B, 8 in case of
+       64B/66B
+   * - TPL_DATA_PATH_WIDTH
+     - Data path width in bytes towards transport layer. Must be greater or
+       equal to ``DATA_PATH_WIDTH``. Must be a power of 2 integer multiple of
+       the F parameter.
+   * - ASYNC_CLK
+     - Set this parameter to 1 if the link clock and the device clocks have
+       different frequencies, or if they have the same frequency but a
+       different source. If set, synchronizing logic and a gearbox of ratio
+       ``DATA_PATH_WIDTH``:``TPL_DATA_PATH_WIDTH`` is inserted to do the rate
+       conversion. If not set, ``TPL_DATA_PATH_WIDTH`` must match
+       ``DATA_PATH_WIDTH``, the same clock must be connected to ``clk`` and
+       ``device_clk`` inputs.
+
+Signal and Interface Pins
+-------------------------
+
+.. hdl-interfaces::
+    :path: library/jesd204/axi_jesd204_tx
+
+    * - s_axi_aclk
+      - All ``S_AXI`` signals and ``irq`` are synchronous to this clock.
+    * - s_axi_aresetn
+      - Resets the internal state of the peripheral.
+    * - S_AXI
+      - Memory mapped AXI-lite bus that provides access to modules register map.
+    * - irq
+      - Interrupt output of the module. Is asserted when at least one of the
+        modules interrupt is pending and enabled.
+    * - clk
+      - `Link clock <jesd204_glossary#clocks>`__ for the JESD204 interface. Must
+        be line clock / 40 for correct operation in 8B/10B mode, line clock /
+        66 in 64B/66B mode.
+    * - reset
+      - Reset active high synchronous with the `Link clock
+        <jesd204_glossary#clocks>`__.
+    * - device_clk
+      - `Device clock <jesd204_glossary#clocks>`__ for the JESD204 interface.
+         Its frequency must be link clock \* ``DATA_PATH_WIDTH`` /
+         ``TPL_DATA_PATH_WIDTH``
+    * - device_reset
+      - Reset active high synchronous with the `Device clock
+        <jesd204_glossary#clocks>`__.
```

Comment:
Please import [jesd204/jesd204_glossary](https://wiki.analog.com/resources/fpga/peripherals/jesd204/jesd204_glossary) and fix all broken references related to this.
On the compile log, it shows as:
```
/hdl/docs/library/jesd204/axi_jesd204_tx/index.rst:: ERROR: Anonymous hyperlink mismatch: 1 references but 0 targets.
/hdl/docs/library/jesd204/axi_jesd204_tx/index.rst:: ERROR: Anonymous hyperlink mismatch: 1 references but 0 targets.
```